### PR TITLE
FES Next.js 13 App Router

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes #[DOCO-1070](https://getpantheon.atlassian.net/browse/DOCO-1070)

## Summary

**[Known Issues](https://docs.pantheon.io/guides/decoupled/overview/troubleshooting#known-issues)** - Updates documentation to note that the Front-End Sites cache purging approach is not currently compatible with the Next.js 13 App Router.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

[DOCO-1070]: https://getpantheon.atlassian.net/browse/DOCO-1070?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: source/content/guides/decoupled/overview/02-considerations.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ You can use frontend frameworks other than Gatsby and Next.js, however the proce
 - A repo cannot be connected to more than one Front-End Site
 - There are known issues around disconnecting and reconnecting a repo
 - There are known issues around the GitHub app and org level permissions
+- The Front-End Sites cache purging approach is not currently compatible with the Next.js 13 App Router.
```
Comment (backlineint) [inline source/content/guides/decoupled/overview/02-considerations.md:68]: I'm not sure someone landing on this page would have enough context to know what 'The Front-End Sites cache purging approach' is. But I struggle with the right way to elaborate on this. How about:

"The caching recommendations outlined for use with our Next.js starter kits are currently only compatible with the Next.js Pages Router, not the App Router."

**Review by backlineint — CHANGES_REQUESTED** [review]

**Comment by whitneymeredith** [comment]
[Considerations Preview](https://pr-8578-documentation.appa.pantheon.site/guides/decoupled/overview/considerations#known-issues-and-limitations)
[Troubleshooting Preview](https://pr-8578-documentation.appa.pantheon.site/guides/decoupled/overview/troubleshooting#known-issues)
[WP Next Preview](https://pr-8578-documentation.appa.pantheon.site/guides/decoupled/wp-nextjs-frontend-starters/caching#using-the-starter-kit)
[Drupal Next Preview](https://pr-8578-documentation.appa.pantheon.site/guides/decoupled/drupal-nextjs-frontend-starters/caching#using-the-starter-kit)

**Review by backlineint — APPROVED** [review]

**Comment by whitneymeredith** [comment]
Thanks @backlineint !

**Review by wordsmither — APPROVED** [review]
lgtm

**Pull request merged**